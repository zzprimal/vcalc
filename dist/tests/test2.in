print(21834);
vector D91eXMo = 40997 - 8008..8015 - 12550..12699 + 8855;
if (36084)
	D91eXMo = D91eXMo / 22646 < 42246 + 34334;
	D91eXMo = D91eXMo;
	vector XQmZy5Rhk4O07m = D91eXMo;
fi;
print(0..221 < D91eXMo - D91eXMo - 21639);
vector nYB4i6sAVUd = 0..132 < 30289;
nYB4i6sAVUd = 10415 * nYB4i6sAVUd;

//CHECK_FILE:./test2.out
