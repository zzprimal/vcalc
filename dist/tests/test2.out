21834
[0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0]
